smart-contracts/wasm-chain-integration/Cargo.lock

```diff
@@ -134,7 +134,7 @@ dependencies = [
  "borsh-schema-derive-internal",
  "proc-macro-crate 0.1.5",
  "proc-macro2",
- "syn",
+ "syn 1.0.107",
```

Comment:
Why does this one still have v1?

Reply:
I noticed this as well, it stems from other dependencies, like rust_decimal (serde_derive), thiserror and num_enum